includes/class-amp-theme-support.php

```diff
@@ -752,7 +752,9 @@ public static function start_output_buffering() {
 	 * @see AMP_Theme_Support::start_output_buffering()
 	 */
 	public static function finish_output_buffering() {
-		echo self::prepare_response( ob_get_clean() ); // WPCS: xss ok.
+		$output = self::prepare_response( ob_get_clean() );
+		AMP_Validation_Utils::add_header();
```

Comment:
I'm going to remove thus until we have it being utilized in the plugin.

Reply:
Sure, that's fine.